docs/administering/ssl.md

```diff
@@ -1,29 +1,38 @@
 ## Sandstorm and HTTPS
 
-If you are using a hostname like `example.sandcats.io`, then you likely already have working HTTPS
-(SSL) for your hostname. This page provides details on a variety of options for setting up HTTPS,
-including the `sandcats.io` free certificates.
+Sandstorm can now terminate TLS connections for most conventional configurations you may wish to employ.
+This includes automatic certificate renewal of certificates, if you utilize a supported DNS provider, as well
```

Comment:
nitpick: "utilize" is an unnecessarily big word for "use," and IMO it scans poorly here. Same thing in another place or two below (I'm not too inclined to die on this hill though...).

Reply:
I also feel the first comma here is extraneous. And so shortening the sentence fragment I wish to merge into the other sentence fragment is nice.